src/main/resources/com/hcl/appscan/jenkins/plugin/auth/ASoCCredentials/config.jelly

```diff
@@ -16,5 +20,9 @@ LICENSE: Apache License, Version 2.0 https://www.apache.org/licenses/LICENSE-2.0
 	<f:entry title="${%label.label}" field="description">
 		<f:textbox />
 	</f:entry>
+
+	<f:entry field="certificates">
```

Comment:
Similar to other comments, I think a field name like "acceptssl" or "acceptInvalidCerts" would be better here.

Reply:
Now, I am using "acceptInvalidCerts" instead of the earlier one.